frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -4,10 +4,11 @@
 ## Editor
 ## Allows contributors to modify or propose a translation
 
-entitydetails-editor-button-copy = Copy
-entitydetails-editor-button-clear = Clear
-entitydetails-editor-button-save = Save
-entitydetails-editor-button-suggest = Suggest
+editor-editor-sign-in-to-translate = <a>Sign in</a> to translate.
+editor-editor-button-copy = Copy
+editor-editor-button-clear = Clear
+editor-editor-button-save = Save
+editor-editor-button-suggest = Suggest
```

Comment:
`editor-editor` looks weird... Is that because you're simply following the `$module-$component` convention?

Reply:
Yeah, I don't like it much either. At some point we should have a conversation about those IDs, and decide on a clear rule.

Reply:
I filed https://bugzilla.mozilla.org/show_bug.cgi?id=1528926 to kick off the conversation about the id naming scheme.